src/components/Select/Select.scss

```diff
@@ -150,6 +150,12 @@ $stacking-order: (
   }
 }
 
+.requiredIndicator:after {
+  content: "*";
+  color: red;
+  margin-left: 4px;
+}
```

Comment:
Should we move that into `src/styles/foundation/_typography.scss` and transform it as a mixin? so `TextField` and `Select` could re-use it.